app/components/failed_network_action/failed_network_action.js

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {Text, View} from 'react-native';
+import {intlShape} from 'react-intl';
+import Button from 'react-native-button';
+
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+import Cloud from './cloud';
+
+export default class FailedNetworkAction extends PureComponent {
+    static propTypes = {
+        actionText: PropTypes.string,
+        errorMessage: PropTypes.string,
+        errorTitle: PropTypes.string,
+        isLandscape: PropTypes.bool.isRequired,
+        onRetry: PropTypes.func.isRequired,
+        theme: PropTypes.object.isRequired,
+    };
+
+    static contextTypes = {
+        intl: intlShape.isRequired,
+    };
+
+    static defaultProps = {
+        showAction: true,
+    };
```

Comment:
AFAICT, this isn't used anywhere. Should we remove?

Reply:
if no longer used we can remove as well as remove the props set when referencing the component